less/v2/apps.less

```
@@ -760,27 +760,80 @@
     position: relative;
     overflow: visible;
     justify-content: flex-end;
-    margin-block-end: calc(var(--font-size-24) + 12px);
+    margin-block-end: calc(var(--font-size-24) + 6px);
     background: none;
 
     h1 {
       position: absolute;
-      inset-block-start: calc(var(--header-height) - 4px);
+      inset-block-start: calc(var(--header-height) - 20px);
       inset-inline: 0;
       color: var(--color-text-dark-primary);
       font-family: var(--dnd5e-font-roboto-slab);
       font-weight: bold;
       font-size: var(--font-size-24);
       text-align: center;
+      pointer-events: none;
```

This does greatly reduce the draggable area of the header. I tested using a `z-index` instead and that allowed continued interaction with the header buttons while still allowing the expanded drag area.

```suggestion
      z-index: -1;
```